### PR TITLE
fit video on phone

### PR DESCRIPTION
**Pull request by BellaChloe**
I was in charge to adapt Youtube videos in a phone format. I modified the Markdown.vue to make it happens.

### CODE REVIEW

File: src/components/generics/Markdown.vue [hunk 1]
```diff
@@ -321,4 +321,13 @@ div[c2c\:role='video'] {
     }
   }
 }
+
+@media screen and(max-width: 844px) {
```
Comment (brunobesson) [inline src/components/generics/Markdown.vue:325]: Where does this value `884` come from? We use bulma breakpoints to distinguish between phone, tablet, desktop and widescreen (768, 1023, 1215 ,1407, +- 1px depending on comparison made)

**Review by brunobesson — DISMISSED** [review]
Thanks for the PR.

Could you please add some description to your PR? What it intends to do, and how? I can guess, but it would ease the discussion a lot.

File: src/components/generics/Markdown.vue (hunk 1 above)
Comment (BellaChloe) [reply src/components/generics/Markdown.vue]: When I put $phone on my doc, it has got an error and I wasn't able to do my internship mission correctly.In school, I learnt to add some value in this case. On the inspector,it was written this value, I wrote this value and the error disappeared.

**Review by cbeauchesne — CHANGES_REQUESTED** [review]
We just met with Chloé. So : 

* fix the error (`phone` variable does not exists)
* Use a simpler way to achieve that
* include the diff in existing CSS descriptors

**Review by cbeauchesne — APPROVED** [review]

**Pull request merged**